Lesson33/Selcan/lesson33-app/src/Exercise4/index.jsx

```diff
@@ -0,0 +1,24 @@
+// Exercise 4: Event Handler with Parameters
+// Create a list of items. When a list item is clicked, display an alert showing the text of the clicked item.
+    
+import { useState } from "react";
+
+export const Exercise4 = () => {
+  const [items, setItems] = useState(['Item 1', 'Item 2', 'Item 3']);
```

Comment:
In this case, you don't really need to use useState hook to store items. The reason for it is that you do not modify the items. 
It's not a mistake but an unnecessary complexity.